CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-## UNRELEASED
+## v0.20.1 (July ??, 2019)
+
+BUG FIXES:
+
+* Fixed issue with templates not rendering with `-once` [[GH-1227] 	(https://github.com/hashicorp/consul-template/pull/1227)], [[GH-1196] 	(https://github.com/hashicorp/consul-template/issues/1196)], [[GH-1207](https://github.com/hashicorp/consul-template/issues/1207)]
```

Comment:
I had included some stray whitespace in my suggested snippet. Fixed here to fix the links for those:

```suggestion
* Fixed issue with templates not rendering with `-once` [[GH-1227](https://github.com/hashicorp/consul-template/pull/1227)], [[GH-1196](https://github.com/hashicorp/consul-template/issues/1196)], [[GH-1207](https://github.com/hashicorp/consul-template/issues/1207)]
```

Also, the other bug fixes could be updated to match.